File: Find-the-Difference.cpp
class Solution 
{
public:
    char findTheDifference(string s, string t) 
    {
        int sums = 0, sumt = 0;
        for (int i = 0; i < s.size(); i++)
         {
            sums += s[i];
        }
        for (int i = 0; i < t.size(); i++) 
        {
            sumt += t[i];
        }
        return (char)(sumt - sums);
    }
};

File: Find-the-Difference.py
class Solution(object):
    def findTheDifference(self, s, t):
        \\\
        :type s: str
        :type t: str
        :rtype: str
        \\\
        sums = 0
        sumt = 0
        for i in range(len(s)):
            sums += ord(s[i])
        for i in range(len(t)):
            sumt += ord(t[i])
        return chr(sumt - sums)